swarm/api/http/server.go

```diff
@@ -60,6 +62,8 @@ var (
 	getListFail     = metrics.NewRegisteredCounter("api.http.get.list.fail", nil)
 )
 
+const SwarmTagHeaderName = "x-swarm-tag"
```

Comment:
then use this constant everywhere

Reply:
can't because of cyclic dependencies between `api` and `client` libraries. we should consider pulling this type of constants to a separate package such as what anton did with the timeouts. in the meanwhile leaving this here.

Reply:
done :+1: